test/ClangImporter/experimental_diagnostics_no_noise.swift

```diff
@@ -0,0 +1,78 @@
+
+import IncompleteTypes
+import cfuncs
+import macros
+import ctypes
+
+// CHECK-NOT: warning
+// CHECK-NOT: error
+let bar = Bar()
+_ = bar.methodReturningForwardDeclaredInterface()
+// CHECK: IncompleteTypes.h:{{[0-9]+}}:{{[0-9]+}}: warning: method 'methodReturningForwardDeclaredInterface' not imported
+// CHECK-NEXT: - (ForwardDeclaredInterface *) methodReturningForwardDeclaredInterface;
+// CHECK-NEXT: ^
+// CHECK-NEXT: IncompleteTypes.h:{{[0-9]+}}:{{[0-9]+}}: note: return type not imported
+// CHECK-NEXT: - (ForwardDeclaredInterface *) methodReturningForwardDeclaredInterface;
+// CHECK-NEXT: ^
+// CHECK-NEXT: IncompleteTypes.h:{{[0-9]+}}:{{[0-9]+}}: note: interface 'ForwardDeclaredInterface' is incomplete
+// CHECK-NEXT: - (ForwardDeclaredInterface *) methodReturningForwardDeclaredInterface;
+// CHECK-NEXT: ^
+// CHECK-NEXT: IncompleteTypes.h:{{[0-9]+}}:{{[0-9]+}}: note: interface 'ForwardDeclaredInterface' forward declared here
+// CHECK-NEXT: @class ForwardDeclaredInterface;
+// CHECK-NEXT: ^
+// CHECK-NEXT: experimental_diagnostics_no_noise.swift:{{[0-9]+}}:{{[0-9]+}}: error: value of type 'Bar' has no member 'methodReturningForwardDeclaredInterface'
+// CHECK-NEXT: _ = bar.methodReturningForwardDeclaredInterface()
+// CHECK-NEXT:     ~~~ ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// CHECK-NOT: warning
+// CHECK-NOT: error
+
+// CHECK-NOT: warning
+// CHECK-NOT: error
+unsupported_parameter_type(1,2)
+// CHECK:      cfuncs.h:{{[0-9]+}}:{{[0-9]+}}: warning: function 'unsupported_parameter_type' not imported
+// CHECK-NEXT:      int unsupported_parameter_type(int param1, _Complex int param2);
+// CHECK-NEXT: {{^}}^
+// CHECK-NEXT: cfuncs.h:{{[0-9]+}}:{{[0-9]+}}: note: parameter 'param2' not imported
+// CHECK-NEXT:      int unsupported_parameter_type(int param1, _Complex int param2);
+// CHECK-NEXT: {{^}}                                           ^
+// CHECK-NEXT: cfuncs.h:{{[0-9]+}}:{{[0-9]+}}: note: built-in type 'Complex' not supported
+// CHECK-NEXT:      int unsupported_parameter_type(int param1, _Complex int param2);
+// CHECK-NEXT: {{^}}                                           ^
+// CHECK-NEXT: experimental_diagnostics_no_noise.swift:{{[0-9]+}}:{{[0-9]+}}: error: cannot find 'unsupported_parameter_type' in scope
+// CHECK-NEXT: unsupported_parameter_type(1,2)
+// CHECK-NEXT: ^~~~~~~~~~~~~~~~~~~~~~~~~~
+// CHECK-NOT: warning
+// CHECK-NOT: error
+
+// CHECK-NOT: warning
+// CHECK-NOT: error
+_ = FUNC_LIKE_MACRO()
+// CHECK:      macros.h:{{[0-9]+}}:{{[0-9]+}}: error: macro 'FUNC_LIKE_MACRO' not imported: function like macros not supported
+// CHECK-NEXT:      #define FUNC_LIKE_MACRO() 0
+// CHECK-NEXT: {{^}}        ^
+// CHECK-NEXT: experimental_diagnostics_no_noise.swift:{{[0-9]+}}:{{[0-9]+}}: error: cannot find 'FUNC_LIKE_MACRO' in scope
+// CHECK-NEXT: _ = FUNC_LIKE_MACRO()
+// CHECK-NEXT:     ^~~~~~~~~~~~~~~
+// CHECK-NOT: warning
+// CHECK-NOT: error
```

Comment:
Not needed.

Reply:
Just for future reference (you already check-not this above). This is fine for now, though. I want to land this 😁